hyperspace/cosmos/Cargo.toml

```diff
@@ -29,6 +29,12 @@ tonic = { version = "0.8", features = ["tls", "tls-roots"] }
 bech32 = "0.9.1"
 bip32 = "0.4.0"
 ed25519-zebra = { version = "3.1.0" }
+tiny-bip39 = "1.0.0"
```

Comment:
are you sure we need all these dependencies?

Reply:
The ``data-encoding`` and ``secp256k1 `` one is redundant. Sorry for the mistake